index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Exo+2&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="assets/styles.css" />
+    <script type="text/javascript" src="script.js" async></script>

```

Comment:
If you work in script with DOM better to move it to the end of the body. Read about it [learjavascript](https://learn.javascript.ru/script-async-defer)

Reply:
In the beginning i had this part in correct place, but then I thought it can be misunderstood by cross-chekers hence I moved it in head)) But now I have fixed.
https://github.com/BlackTruth/virtual-keyboard/blob/6297a0817c03affb1948c9e73a6352687e6ff449/dist/index.html#L13-L15